apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktService.kt

```diff
@@ -242,20 +243,20 @@ class AktivitetspliktService(
         }
     }
 
-    fun opprettAktivitetsgradForOppgave(
+    fun upsertAktivitetsgradForOppgave(
         aktivitetsgrad: LagreAktivitetspliktAktivitetsgrad,
         oppgaveId: UUID,
         sakId: SakId,
         brukerTokenInfo: BrukerTokenInfo,
-    ) {
+    ): AktivitetspliktVurdering {
         val kilde = Grunnlagsopplysning.Saksbehandler.create(brukerTokenInfo.ident())
-        require(
-            aktivitetspliktAktivitetsgradDao.hentAktivitetsgradForOppgave(oppgaveId).isEmpty(),
-        ) { "Aktivitetsgrad finnes allerede for oppgave $oppgaveId" }
         sjekkOmAktivitetsgradErGyldig(aktivitetsgrad)
-        aktivitetspliktAktivitetsgradDao.opprettAktivitetsgrad(aktivitetsgrad, sakId, kilde, oppgaveId)
+        aktivitetspliktAktivitetsgradDao.upsertAktivitetsgradForOppgave(aktivitetsgrad, sakId, kilde, oppgaveId)
         val oppgave = oppgaveService.hentOppgave(oppgaveId)
-        runBlocking { sendDtoTilStatistikk(sakId, brukerTokenInfo, behandlingId = UUID.fromString(oppgave.referanse)) }
+
+        runBlocking { sendDtoTilStatistikk(sakId, brukerTokenInfo, null) }
```

Comment:
Hvorfor er ikke dette på kafka så vi slipper en runBlocking her

Reply:
Åja ser nå at det er et eventyr at dilldall nedover der som ikke helt reflekterer hva den gjør, skummelt at den skal kanskje tryne på det etter at den egentlig er ferdig men vi kan se på en måte for å gjøre den mer sane på

Reply:
Ta en runde på det etter denne pren?